iib/web/models.py

```diff
@@ -1239,7 +1317,8 @@ def from_json(cls, kwargs, batch=None):
             raise ValidationError('"organization" must be a string')
 
         # Validate from_bundle_image is correctly provided
-        from_bundle_image = request_kwargs.pop('from_bundle_image')
+        # TODO change pop for get, as It would make type more strict
+        from_bundle_image = request_kwargs.get('from_bundle_image')
```

Comment:
Parameter from_bundle_image is required in the IIB docs, and so It is not allowed by mypy to have an empty value in that parameter (In case we want to work with just one typeDict - what makes in my opinion sense}. Also, we are replacing this value right away, so I do not think this is a problem.